src/config/PrinterList.json

```diff
@@ -2,11 +2,11 @@
   "printers": [
     {
       "name": "stub",
-      "hideInProduction": true
+      "nonProductionPrinter": true
```

Comment:
Out of interest, why this name change?  I find it quite hard to digest `nonProductionPrinter` for some reason.  I think it's because it's a negation of a boolean.  Is there a way you can make it `"productionPrinter": false` instead?